.travis.yml

```diff
@@ -142,22 +140,15 @@ install:
       cp "$HOME/hombebrew_cache/libtorrent-rasterbar-1.0.11+git20172003.8736a59adc.el_capitan.bottle.tar.gz" "$(brew --cache)"
       brew install "$HOME/hombebrew_cache/libtorrent-rasterbar.rb"
 
+      brew install qt
+      brew link --force qt
```

Comment:
I think these lines should move up and group with `brew install colormake ccache zlib`